src/operator/tensor/la_op.cc

```diff
@@ -251,42 +253,42 @@ NNVM_REGISTER_OP(_backward_linalg_potri)
 .set_attr<nnvm::TIsBackward>("TIsBackward", true)
 .set_attr<FCompute>("FCompute<cpu>", LaOpBackward<cpu, 2, 2, 3, 1, potri_backward>);
 
-NNVM_REGISTER_OP(_linalg_trmm)
+NNVM_REGISTER_OP(linalg_trmm)
```

Comment:
All these operators are now registered with an initial underscore followed by an alias for backward compatibility.

Reply:
OK, I will do this